composer.json

```diff
@@ -9,6 +9,7 @@
         "drupal/webform": "^6.0",
         "itk-dev/drupal_psr6_cache": "^1.0",
         "itk-dev/openid-connect": "^3.1",
+        "os2web/os2web_audit": "^0.1",
```

Comment:
Why is this added in a pull request related to keys?

Reply:
A duplicate `require` and `require-dev` is added to `composer.json` in https://github.com/itk-dev/os2forms_nemlogin_openid_connect/pull/20. This was somehow missed by myself in a review.

The duplicated `require` contained `"os2forms/os2forms": "^3.17"`, in which the `os2web/os2web_audit` requirement came from. However, the only thing needed from os2forms is os2web_audit.

Reply:
But why do we need “audit” in a pull request related to "key"?

Reply:
We don't - we just need to not remove it. 

https://github.com/itk-dev/os2forms_nemlogin_openid_connect/pull/20 mistakenly adds an extra `require`  and `require-dev` section before the original one. This newly added `require` section is missing the `os2forms` requirement that the original one had.

I suppose your right in that this change from a full `os2forms` requirement to a direct `audit` requirement could have it's own PR. I will update with the original requirement.